test/volume_cache_test.js

```diff
@@ -197,36 +200,36 @@ suite('volume cache test', function () {
     var cacheName = 'tmp-obj-dir-' + Date.now().toString();
     var fullPath = path.join(localCacheDir, cacheName);
 
-    var instance1 = yield cache.get(cacheName);
-    var instance2 = yield cache.get(cacheName);
+    var instance1 = await cache.get(cacheName);
+    var instance2 = await cache.get(cacheName);
 
-    yield cache.release(instance1.key);
+    await cache.release(instance1.key);
 
     let futurePurgeDate = new Date();
     futurePurgeDate.setHours(futurePurgeDate.getHours() + 5);
 
     // should remove only instance1
     cache.purge(cacheName, futurePurgeDate);
 
-    var instance3 = yield cache.get(cacheName);
+    var instance3 = await cache.get(cacheName);
     assert.notEqual(instance3.key !== instance1.key);
 
-    yield cache.release(instance2);
-    yield cache.release(instance3);
+    await cache.release(instance2);
+    await cache.release(instance3);
 
     cache.purge(cacheName, futurePurgeDate);
 
-    var instance4 = yield cache.get(cacheName);
+    var instance4 = await cache.get(cacheName);
```

Comment:
I added a bug for these type of changes.  I attempted to not change things if I didn't have to so it could be saved for later cleanup.  https://bugzilla.mozilla.org/show_bug.cgi?id=1346249